File: frontend/src/types/publications.ts
export interface Publication {
  id: number;
  title: string;
  title_en: string;
  authors: string;
  venue: string;
  venue_short: string;
  year: number;
  url: string;
  tags: string[];
}

export interface PublicationsResponse {
  items: Publication[];
}

File: backend/app/models/publications.py
from pydantic import BaseModel
from typing import List, Optional

class Publication(BaseModel):
    id: int
    title: str
    title_en: str
    authors: str
    venue: str
    venue_short: str
    year: int
    url: Optional[str] = ""
    tags: List[str] = []

class PublicationsResponse(BaseModel):
    items: List[Publication]
